License2Deploy/rolling_deploy.py

```diff
@@ -179,6 +179,13 @@ def confirm_lb_has_only_new_instances(self, wait_time=30):
     logging.info("Deployed instances {0} to ELB: {1}".format(instance_ids, lb))
     return instance_ids
 
+  def tag_ami(self, ami_id):
+    ''' Tagging AMI with DEPLOYED tag '''
+    try:
+      self.conn_ec2.create_tags([self.ami_id], {"DEPLOYED": "TRUE"})
```

Comment:
Was is deployed to QA? STG? PRD? Value should probably be a list of environments it has gone to.